Sources/MapboxDirections/Intersection.swift

```diff
@@ -292,10 +317,23 @@ extension Intersection: Codable {
         if let lanes = try container.decodeIfPresent([Lane].self, forKey: .lanes) {
             approachLanes = lanes.map { $0.indications }
             usableApproachLanes = lanes.indices { $0.isValid }
+            preferredApproachLanes = lanes.indices { ($0.isActive ?? false) }
+            
+            let temp = LaneIndication(rawValue: 0)
+            var usableIndications = [LaneIndication]()
+            lanes.forEach { lane in
+                if lane.validIndication != nil {
+                    usableIndications.append(lane.validIndication!)
+                }
+            }
+            usableLaneIndication = usableIndications.reduce(temp) { return $0.union($1) }
```

Comment:
Nit: we can omit the `return` keyword here because the closure contains only a single expression.